packages/capabilities/bundle/config/upstream/tanzu-capabilities-controller-manager.yaml

```diff
@@ -9,6 +9,8 @@ metadata:
     app: tanzu-capabilities-manager
   name: tanzu-capabilities-controller-manager
   namespace: #@ data.values.namespace
+  annotations:
+    kapp.k14s.io/disable-default-label-scoping-rules: ""
```

Comment:
How does this annotation help? Would be great if you could add a brief explanation to the commit description.

Reply:
updated the commit and added a comment [here](https://github.com/vmware-tanzu/tanzu-framework/pull/3523/files#r985302542) to explain